app/Models/Forum/Topic.php

```diff
@@ -811,6 +817,10 @@ public function setIssueTag($tag)
     {
         $this->topic_type = static::typeInt($tag === 'confirmed' ? 'sticky' : 'normal');
 
+        if (in_array($tag, static::PLATFORM_ISSUE_TAGS, true)) {
+            $tag = "osu!{$tag}";
+        }
```

Comment:
probably move this logic to its own method cuz it comes up 3 times (`tagForTopicTitle()` or something)

alternatively include the prefix in each PLATFORM_ISSUE_TAGS